src/sagemaker/serve/builder/jumpstart_builder.py

```diff
@@ -431,6 +431,24 @@ def tune_for_tgi_jumpstart(self, max_tuning_duration: int = 1800):
             sharded_supported=sharded_supported, max_tuning_duration=max_tuning_duration
         )
 
+    def set_deployment_config(self, config_name: Optional[str]) -> None:
+        """Sets the deployment config to apply to the model.
+
+        Args:
+            config_name (Optional[str]):
+                The name of the deployment config. Set to None to unset
+                any existing config that is applied to the model.
+        """
+        self.pysdk_model.set_deployment_config(config_name)
```

Comment:
the `pysdk_model` will be overridden when calling `build()` here https://github.com/aws/sagemaker-python-sdk/blob/6520a2b5d48caa7babd7d5802c1fe461ebb22182/src/sagemaker/serve/builder/jumpstart_builder.py#L470, can we fix it to persist the jumpstart model instance with the applied config?